### PR TITLE
Issue #6 by kumikoono, naoi: Assertions for table rows

### LINKED ISSUE #6: Assertions for table rows
Cloud Orchestrator uses multiple rows with the same text in a table, so we want to specify the row index for search and edit.

### PR DESCRIPTION
**Pull request by kumikoono**
Add assertions related to TableRow

### CODE REVIEW

**Comment by onotmil** [comment]
It looks good to me.

File: src/Drupal/DrupalExtension/MinkAwareTrait.php [hunk 1]
```diff
@@ -169,4 +169,26 @@ public function saveScreenshot($filename = null, $filepath = null)
...
+  /**
+   * Return a region from the current page.
+   *
+   * @throws \Exception
+   *   If region cannot be found.
+   *
+   * @param string $region
+   *   The machine name of the region to return.
+   *
+   * @return \Behat\Mink\Element\NodeElement
```
Comment (naoi) [inline src/Drupal/DrupalExtension/MinkAwareTrait.php]: @kumikoono The comment should be:

```
   * @param string $region
   *   The machine name of the region to return.
   *
   * @return \Behat\Mink\Element\NodeElement
   *
   * @throws \Exception
   *   If region cannot be found.
```

File: src/Drupal/DrupalExtension/MinkAwareTrait.php [hunk 2]
```diff
@@ -169,4 +169,26 @@ public function saveScreenshot($filename = null, $filepath = null)
...
+   */
+  public function getRegion($region)
+  {
```
Comment (naoi) [inline src/Drupal/DrupalExtension/MinkAwareTrait.php]: @kumikoono Should we probably follow Drupal coding standard rule at least for our code that we are adding?

```
  public function getRegion($region) {
```

**Review by naoi — CHANGES_REQUESTED** [review]
@kumikoono Thank you for the PR.  I posted my comments.  Thanks

File: src/Drupal/DrupalExtension/MinkAwareTrait.php (hunk 2 above)
Comment (kumikoono) [reply src/Drupal/DrupalExtension/MinkAwareTrait.php]: Done

File: src/Drupal/DrupalExtension/MinkAwareTrait.php (hunk 1 above)
Comment (kumikoono) [reply src/Drupal/DrupalExtension/MinkAwareTrait.php]: Done

**Pull request merged**